Gemfile

```diff
@@ -19,7 +19,7 @@ gem 'dough-ruby',
   ref: 'cf08913'
 gem 'kaminari'
 gem 'letter_opener', group: :development
-gem 'mas-rad_core'
+gem 'mas-rad_core', github: 'moneyadviceservice/mas-rad_core', branch: 'geocoding'
```

Comment:
This will be reset once https://github.com/moneyadviceservice/mas-rad_core/pull/5 is pulled in.